.gitignore

```diff
@@ -56,3 +56,5 @@ target/
 *~
 *.swp
 site/
+
+.idea
```

Comment:
I am not an IntelliJ IDEA user, but if this line can help other developers, it deserves a place into the default Python `.gitignore` specified by @pchaigno.

In the meantime, we can include the `.idea` to the gitignore if there is absolutely no impact for developers, and this call is up to you @memelet. I do not want a new comer on the project to be stuck finding why it's new addition is not added to git because we specified a behavior. Will a developer want to add `.idea` files to the github repository? If the answer is no then we can merge this line, otherwise we should remove it.